### DISCUSSION TITLE
When i tried to send message from Android mobile , I am getting empty message

### DISCUSSION THREAD

**Discussion by Sandeep-Narahari** [tulir/whatsmeow#368] [General]
`		fmt.Println(len(v.Message.GetConversation()), v.Message.GetConversation())
`
Got the length as 0 , 
Please let me know how to deal with this problem 
Thank you

**Comment by mamur-rezeki** [comment]
Different type of messages has different way to get the `Text`, `Caption` or `Label` . 

i.e. :
- `VideoMessage` has `GetCaption()`
- `ImageMessage` has `GetCaption()`
- `ExtendedTextMessage` has `GetText()`
- `Conversation` has `GetConversation()`

In some case, they're sent using `ExtendedTextMessage`, so `GetConversation()` isn't available.
